src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -182,7 +221,7 @@ private static void UpdateSpan(dynamic controllerContext, Span span)
         {
             try
             {
-                var req = controllerContext?.Request as HttpRequestMessage;
```

Comment:
Yikes, I thought we had removed all uses of `dynamic`. I'll create a ticket in the backlog.

Reply:
You can assign it to me. I already have a wip branch for this work.